deepchem/models/torch_models/normalizing_flows_pytorch.py

```diff
@@ -0,0 +1,55 @@
+"""
+Normalizing flows for transforming probability distributions using PyTorch.
+"""
+
+import logging
+
+import torch
+import torch.nn as nn
+
+
+logger = logging.getLogger(__name__)
+
+
+class Affine(nn.Module):
```

Comment:
Can you add to docs at https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/layers.rst?